pkg/kritis/review/review.go

```diff
@@ -50,42 +50,113 @@ func New(c *Config) Reviewer {
 	}
 }
 
+// Flatten allowlist from a list of generic attestation policies
+func generateGapAllowlist(gaps []v1beta1.GenericAttestationPolicy) []string {
+	allowlist := []string{}
+	for _, gap := range gaps {
+		for _, pattern := range gap.Spec.AdmissionAllowlistPatterns {
+			allowlist = append(allowlist, pattern.NamePattern)
+		}
+	}
+	return allowlist
+}
+
+// reviewImageWithGAP reviews single image against a single generic attestation policy
```

Comment:
Could you please also document what the return values represent?

Reply:
ack.